tests/NSubstitute.Analyzers.Tests.Shared/DiagnosticIdentifierTests/DiagnosticIdentifierTests.cs

```diff
@@ -74,14 +77,22 @@ public void DiagnosticIdentifiers_WithinCategory_ShouldHaveConsecutiveNumbers()
         [Fact]
         public void DiagnosticDescriptors_Categories_ShouldMatchDiagnosticIdentifiers()
         {
-            var diagnosticDescriptors = DiagnosticDescriptors.Select(desc => (DiagnosticDescriptor)desc.GetValue(null)).ToList();
             var descriptionEnumMap = ((DiagnosticCategory[])Enum.GetValues(typeof(DiagnosticCategory))).ToDictionary(value => value.GetDisplayName());
 
-            var invalidCategoriesDescriptor = diagnosticDescriptors.Where(desc => (int)descriptionEnumMap[desc.Category] != GetCategoryId(desc.Id));
+            var invalidCategoriesDescriptor = DiagnosticDescriptors.Where(desc => (int)descriptionEnumMap[desc.Category] != GetCategoryId(desc.Id));
 
             invalidCategoriesDescriptor.Should().BeEmpty("because descriptor category should match identifier category");
         }
 
+        [Fact]
+        public void DiagnosticDescriptors_HelpLinkUri_ShouldPointToProperDiagnosticDocumentation()
+        {
+            var invalidHelpLinkDescriptors = DiagnosticDescriptors
+                .Where(diagnostic => diagnostic.HelpLinkUri != $"https://github.com/nsubstitute/NSubstitute.Analyzers/blob/master/documentation/{diagnostic.Id}.md");
+
+            invalidHelpLinkDescriptors.Should().BeEmpty();
```

Comment:
~Maybe update (or clone) this test to ensure each `DiagnosticDescriptors` item exists as `./documentation/rules/{id}.md` in the file system? Then it makes sure if a diagnostic gets added there will be a prompt to update the documentation.~ Just saw `DocumentationTests`, I think that covers this already 😄